pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -25,28 +25,44 @@ type ServiceBindingRequestSpec struct {
 	// +optional
 	CustomEnvVar []corev1.EnvVar `json:"customEnvVar,omitempty"`
 
+	// NOTE: Services & Applications will be made non-optional
+	// after deprecated fields are removed.
+
+	// Services are used to identify multiple backing services.
+	// +optional
+	Services *[]BackingServiceSelector `json:"services,omitempty"`
+
+	// Applications are used to identify the application connecting to the
+	// backing service.
+	// +optional
+	Applications *[]ApplicationSelector `json:"applications,omitempty"`
+
+	// DetectBindingResources is flag used to bind all non-bindable variables from
+	// different subresources owned by backing operator CR.
+	// +optional
+	DetectBindingResources bool `json:"detectBindingResources"`
```

Comment:
omitempty should be included for `+optional` fiellds

Reply:
Do you mind if I fixed it in a different PR, since the change isn't related?

Reply:
sure